src/App/App.js

```diff
@@ -1,26 +1,39 @@
 import "./App.css";
 import React, { Component } from "react";
-import movieData from "../movieData";
 import MovieContainer from "../MovieContainer/MovieContainer";
 import SpecificMovieCard from "../SpecificMovieCard/SpecificMovieCard";
+import { getData } from "../ApiCalls";
 
 class App extends Component {
   constructor() {
     super();
-    this.state = { movies: movieData.movies, clickedImg: [] };
+    this.state = { movies: [], clickedImg: [] };
   }
 
   getDetails = (event) => {
-    let clicked = this.state.movies.filter(
-      (movie) => movie.id === parseInt(event.target.id)
-    );
-    this.setState({ clickedImg: clicked });
+    let movieId = `/movies/${parseInt(event.target.id)}`;
+    getData(movieId).then(data => {
+      console.log(data);
+      this.setState({ clickedImg: new Array(data[0].movie) })
```

Comment:
Interesting use of `new Array` - you could consider just putting this item into an array like `[data[0].movie]`